conftest.py

```diff
@@ -75,6 +75,40 @@ def fix_missing_hostname():
             logger.error("Failed to add hostname")
 
 
+@pytest.fixture(scope="session", autouse=True)
+@pytest.mark.uncollectif(lambda: version.current_version() < "5.6")
+def patch_appliance():
+    """ Patch appliance over SSH"""
+    logger.info('Patching appliance')
+    # (local_path, remote_path, md5/None) trio
+    patch_args = (
+        (str(patches_path.join('miq_application.js.diff')),
+         '/var/www/miq/vmdb/app/assets/javascripts/miq_application.js',
+         None),
```

Comment:
Yep, thats the idea. We also patch the very beginning of the file where any additions from eng is highly unprobable.